documentation/syslog_ng_destination.md

```diff
@@ -0,0 +1,75 @@
+# syslog_ng_destination
+
+[Back to resource list](../README.md#resources)
+
+## Actions
+
+- `create` - Create a syslog-ng destination configuration file
+- `delete` - Remove a syslog-ng destination configuration file
+
+## Properties
+
+| Name                   | Type          | Default                          | Description                                                         | Allowed Values      |
+| ---------------------- | ------------- | -------------------------------- | ------------------------------------------------------------------- | ------------------- |
+| `config_dir`           | String        | /etc/syslog-ng/destionation.d    | Directory to create configuration file in                           |                     |
+| `cookbook`             | String        | syslog-ng                        | Cookbook to source configuration file template from                 |                     |
+| `template`             | String        | syslog-ng/destination.conf.erb   | Template to use to generate the configuration file                  |                     |
+| `owner`                | String        | root                             | Owner of the generated configuration file                           |                     |
+| `group`                | String        | root                             | Group of the generated configuration file                           |                     |
+| `mode`                 | String        | `'0640'`                         | Filemode of the generated configuration file                        |                     |
```

Comment:
```suggestion
| `mode`                 | String        | `0640`                         | Filemode of the generated configuration file                        |                     |
```

Reply:
We should probably pick chose to either block quite everything or not (I always side with block quoting)

Reply:
Sounds a plan, I'll change them all to block quoted.